manifest.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "freesurfer-recon-all",
-  "label": "FreeSurfer 7.2.0: run recon-all",
+  "name": "freesurfer-recon-all-can",
+  "label": "FreeSurfer 7.2.0: run recon-all CAN",
```

Comment:
I'm assuming that I'll just change these back to flywheel's versions, correct?

Reply:
Yes I suspect you would want to revert those. Let me know if you need me to revert them